plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -477,7 +488,20 @@ SecRule REQUEST_FILENAME "@contains /.well-known/carddav" \
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} PROPFIND'"
 
+# Appearance and accessibility
 
+# Enabling any default theme requires the PUT method.
+# This FP prevents any theme from permanent activation.
```

Comment:
Please edit this comment. Don't write about FPs but the opposite, write what rule does. Anyway, I think the first sentence is enough.

Reply:
Updated.